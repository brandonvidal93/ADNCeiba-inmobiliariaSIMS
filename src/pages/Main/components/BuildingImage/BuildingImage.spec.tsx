import React from 'react';
import { render } from '@testing-library/react';
import { BuildImage } from '.';

describe('Biulding Image Test', () => {
  it('should render building image', () => {
    const { container } = render(<BuildImage cover="build.jpg" />);
    expect(container).toMatchSnapshot();
  });
});