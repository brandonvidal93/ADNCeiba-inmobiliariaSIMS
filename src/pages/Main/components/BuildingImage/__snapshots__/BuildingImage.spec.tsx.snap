// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Biulding Image Test should render building image 1`] = `
<div>
  <img
    alt="Build cover"
    src="build.jpg"
  />
</div>
`;

exports[`Biulding Image Test should render building image with cover is null 1`] = `<div />`;
